documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -325,3 +325,49 @@ withCredentials([string(credentialsId: 'TOKEN_ID', variable: 'secret')]) {
   """
 }
 ```
+
+## Established Jenkins pipelines
+
+### PR
+
+#### Herbert
+
+<img src="./images/08_HerbertPR.svg">
+
+#### Horace
+
```

Comment:
For this Horace PR diagram, a suitable legend could be (in the text) "This diagram shows the process of triggering and running a build of Horace based on the push of a developer's branch to origin and the state of the current  Herbert build". It is worth saying additionally in the text or diagram box if Herbert is based on a current branch or Herbert master.

Herbert appears in two places on this diagram and it may be worth unifying them somehow. Saying explicitly that Herbert<branch>.git is the result of the Herbert build, so "Herbert <branch> build output" rather than the more cryptic .git text might also make it more immediately clear.